### DISCUSSION TITLE
Best-of MkDocs list

### DISCUSSION THREAD

**Discussion by pawamoy** [mkdocs/mkdocs#2823] [Show and tell]
I've just created a best-of list for MkDocs: https://github.com/pawamoy/best-of-mkdocs
The initial contents come from the Plugins wiki page.
Let me know if you find it nice or useless :wink:

**Comment by Andre601** [comment]
I like the idea. Tho, one thing I dislike is that you outright mark projects as "dead" because they haven't received any updates within a year.
Perhaps, the plugin/theme/extension doesn't require any maintenance? It can be that a resource is that simple that it doesn't require any updates whatsoever, so marking them as "dead" is IMO bad.

**Reply by pawamoy** [reply]
You're right! I'll see if there's a configuration option to either change the text and icon ("dead", skull icon to something lighter), or to completely remove these markers (I didn't write the generator).

**Reply by Andre601** [reply]
If there isn't such a thing should it perhaps be suggested to the creators of the action.

**Reply by pawamoy** [reply]
Would this be better?

```yaml
  project_inactive_months: 12
  project_dead_months: 99999  # "dead" is a somewhat strong assumption, so we avoid making it
```

I don't think "inactive" holds a strong meaning like "dead", but I still increased the threshold.

**Reply by Andre601** [reply]
This is up to you on how you want to handle this.

I made an issue on the generator repo to discuss this and propose some settings to add: https://github.com/best-of-lists/best-of-generator/issues/56

**Comment by oprypin** [comment]
Thanks!

* I replaced the Plugins page to just link to best-of-mkdocs instead
* I edit the documentation to reflect the same: https://github.com/mkdocs/mkdocs/pull/3207

**Reply by Andre601** [reply]
Not sure if the wiki edit was really needed.

**Reply by pawamoy** [reply]
Since we included a link to the best-of list in MkDocs' own documentation, we had [only one contribution](https://github.com/mkdocs/best-of-mkdocs/pull/39) to the list. Meanwhile the wiki had many more (not a crazy amount, but still more). We do want to direct users to the best-of list, to incite them to contribute their plugins/themes/extensions to the list directly rather than in the wiki, because the list provides formatted data that can be used by the new `get-deps` command implemented by @oprypin. The more exhaustive the list is, the better. Also the list is more maintainable than the wiki: the CI logs show us when a repository has moved (changed owner, renamed, etc.).
